notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1120_az_brake_analysis.ipynb

```diff
@@ -0,0 +1,581 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def addTime(t_start, t_end):
</pre><p>Function names should be <code>snake_case</code> instead of <code>semiCamelCase</code>. Could you update the names of functions here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Names updates</p>